docs/BuildInstructions.md

```diff
@@ -0,0 +1,17 @@
+# Build and Deploy Instructions for Mekanism #
```

Comment:
This file is unnecessary in my opinion. It follows standard for mod build steps. The forge or MDK instructions can be linked instead.

Reply:
I think it is.  I'm looking for instructions that a smart 8 year could follow.  
The forge instructions I could find, but not the MDK instructions.
In my mind, we're setting up an on-ramp that will allow newbies (both to minecraft and open source) to build and use the product.

That requires step by step, mekanism specific instructions.

Reply:
The default setup steps should cover 80%
https://mcforge.readthedocs.io/en/latest/gettingstarted/#from-zero-to-modding

Reply:
Personally I think that's setting the bar a bit low as minecraft modding should be done with a moderate to high level understanding of Java and how to work out how the code base functions on one's own.

Modding from scratch would be a far better target for a young audience.

Reply:
@thommy101 so are the Forge instructions and the MDK instructions the same thing?
If so, what does MDK stand for.

Reply:
@thiakil You're right, it is setting the bar low - for the would-be-developer.  I think it sets a high bar for a document maintainer.

I am of the opinion that the lower the bar to entry can be set, the more people will be able to  work on the code, the better the quality and quantity of the changes being made.

What are the benefits that we get from setting the bar at a particular level, 
and how do we communicate that bar level to a new comer?

When you say "Modding from scratch" were you referring to this post?
https://www.minecraftforum.net/forums/archive/tutorials/930698-forge-modding-from-scratch-3-27-2012